bran/helpers/blender.py

```diff
@@ -0,0 +1,102 @@
+from bran.helpers.utils import get_security_groups, get_local_awsconfig, list_to_choices
+import os
+
+def get_blender_questions():
+    """
+    Constructs and returns questions for cli to set up ec2 instance properties for image generation with Blender.
+
+    Return:
+        questions ([dicts]): a list of dictionaries with each dictionary representing
+            a different question
+    """
+
+    amis = ['Blender:ami-0c66a2d9734c3f1aa', "Blender-GPU:ami-0e96d01a5638bccd8"]
+    instance_types = ['t2.large', 'g4dn.xlarge','g3.4xlarge', 't2.medium','t2.micro']
+    sg_names = get_security_groups()
+    
+    questions = [
+        {
+            'type': 'list',
+            'name': 'ami',
+            'message': 'Select AMI',
+            'choices': amis
+        },
+        {
+            'type': 'list',
+            'name': 'instance',
+            'message': 'Select Instance Type',
+            'choices': instance_types
+        },
+        {
+            'type': 'checkbox',
+            'name': 'sg',
+            'message': 'Select all Security Groups',
+            'choices': list_to_choices(sg_names)
+        },
+        {
+            'type': 'input',
+            'name': 'storage',
+            'message': 'Enter storage amount (gb)'
+        },
+        {
+            'type': 'input',
+            'name': 'model',
+            'message': 'Enter filepath of desired blender model',
+            'validate': lambda p: os.path.isfile(p)
+        },
+        {
+            'type': 'input',
+            'name': 'script',
+            'message': 'Enter filepath of generation script',
+            'validate': lambda p: os.path.isfile(p)
+        },
+        {
+            'type': 'input',
+            'name': 'requirements',
+            'message': 'Enter filepath of requirements.txt file',
+            'validate': lambda t: os.path.isfile(t) 
+        }
+    ]
+    return questions
+
+
+
+def get_blender_init_script(script_name):
```

Comment:
kind of a nit, but curious if it would be possible to store this script as its own `.sh` or `.bash` file and then read it in with this function, rather than storing it as one big string directly in the code? It would make maintaining the script easier since you would get actual syntax highlighting, etc when editing it.

For the formatting part, you can accomplish the same thing by placing specific identifiers into the separate script (like `<key_id>`) and then replacing them in the loaded string. This is done in the BBOX plugin, see here: https://github.com/autognc/ravenML-train-plugins/blob/master/rmltraintfbbox/rmltraintfbbox/utils/helpers.py#L129

definitely not critical, but could be nice

Reply:
I like this idea. definitely doable and shouldn't be hard to implement.